launch/dynamo-run/src/input/batch.rs

```diff
@@ -109,6 +111,11 @@ pub async fn run(
     tracing::info!("Timer start.");
     let start = Instant::now();
     let mut lines = buffered_input.lines();
+    let template: Option<&'static RequestTemplate> = template.map(|t| {
+        let boxed = Box::new(t);
+        let leaked = Box::leak(boxed) as &'static RequestTemplate;
+        leaked
+    });
```

Comment:
Could you use an `Arc` here instead? The cost is the same because `Arc` only takes the log to increment/decrement the ref count (so on clone and drop), but I think it will be easier to understand in the future.

Reply:
Make sense, will fix this